addons/sms/tests/test_sms_template_access_rights.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import SavepointCase, new_test_user
+from odoo.exceptions import AccessError
+
+
+class TestSmsTemplateAccessRights(SavepointCase):
```

Comment:
I would add a post-install tag to make sure it runs with all modules installed.

Reply:
done